vertexerror.go

```diff
@@ -0,0 +1,60 @@
+package anthropic
+
+import (
+	"fmt"
+)
+
+// VertexAPIError provides error information returned by the Anthropic API.
+type VertexAPIError struct {
+	Code    int                     `json:"code"`
+	Message string                  `json:"message"`
+	Status  string                  `json:"status"`
+	Details []VertexAPIErrorDetails `json:"details"`
+}
+
+type VertexAPIErrorDetails struct {
+	Type     string `json:"@type"`
+	Reason   string `json:"reason"`
+	Metadata struct {
+		Method  string `json:"method"`
+		Service string `json:"service"`
+	} `json:"metadata"`
+}
+
+/*
```

Comment:
Please delete these if they are not used.

Reply:
Done